### ISSUE TITLE
Question about frozen encoder

### ISSUE THREAD

**Issue by woshitiancai321** [tianrun-chen/SAM-Adapter-PyTorch#72]
Great work! I didn't notice any code for detaching gradients (e.g., grad.detach(), nograd) in the image encoder during training.     But I found you claim frozen encoder, its make me confusing. can you tell me some details about it?    thanks ~~~~

**Comment by woshitiancai321** [comment]
Sorry  i found require grad in code   thanks
